query/mgr_test.go

```diff
@@ -1131,6 +1145,15 @@ func setupQueryManagersWithClusterVersionProvider(numMgrs int, numPartitions int
 	}
 }
 
+func deleteQuery(t *testing.T, query string, ctx *mgrCtx) {
+	ast, err := parser.NewParser(nil).ParseTSL(query)
+	require.NoError(t, err)
+	for _, pair := range ctx.qms {
+		err = pair.qm.DeleteQuery(*ast.DeleteQuery)
+		require.NoError(t, err)
```

Comment:
Leyt's add a check here that the query has actually been deleted. I.e. check that mgr.GetPreparedQueryParamSchema return _, false

Reply:
makes sense. added.